src/Console/ConsoleOutput.php

```diff
@@ -87,6 +88,28 @@ public function logBadMsiErrorMessage(MetricsCalculator $metricsCalculator, floa
         );
     }
 
+    public function logMinMsiCanGetIncreasedNotice(MetricsCalculator $metricsCalculator, float $minMsi, string $type): void
+    {
+        $typeString = ($type === TestRunConstraintChecker::MSI_FAILURE ? 'MSI' : 'Covered Code MSI');
```

Comment:
Probably cleaner to check if it is 

MSI or covered MSI, and throw an error otherwise

Reply:
Done. I also decided to get rid of the double check `$type === TestRunConstraintChecker::MSI_FAILURE`